.github/workflows/tests.yaml

```diff
@@ -29,6 +29,13 @@ jobs:
           --health-timeout 5s
           --health-retries 5
     steps:
+      - name: Install latest sbom-scorecard
+        run: |
+          mkdir ~/.sbom_scorecard
+          wget -O ~/.sbom_scorecard/scorecard $(curl -s https://api.github.com/repos/ebay/sbom-scorecard/releases/latest | 
+          grep -m 1 "http.*linux-amd64" |
+          awk '{print $2}' | sed 's|[\"\,]*||g')
+          export SBOM_SCORECARD="~/.sbom_scorecard/scorecard"
```

Comment:
Also let's install it in `/usr/local/bin` . Downloading it to `~/sbom_scorecard` (doesnt need to be a hidden folder) is fine, but it should be copied or symlinked to `/usr/local/bin`